src/utility/arrayUtils.js

```diff
@@ -1,5 +1,4 @@
-// Arrayextensions re-imagined as a utility object.
-var arrayUtils = arrayUtils || {};
+import {Creature} from "../creature";
```

Comment:
Consistency, all your other imports are `{ Creature }`

Reply:
Yeah this is the reason why we need to get a style guide. So ESLint can detect this kind of stuff.